gradle/libs.versions.toml

```diff
@@ -9,7 +9,7 @@ aopalliance="1.0"
 zipkinAws="1.2.5"
 logback = "1.5.7"
 wavefrontSdk="3.4.3"
-otelInstrumentationBomAlpha="1.33.6-alpha" # Brings in io.opentelemetry:opentelemetry-bom:1.41.0
+otelInstrumentationBomAlpha="2.7.0-alpha" # Brings in io.opentelemetry:opentelemetry-bom:1.41.0
```

Comment:
I guess we'll have to manually update this comment each time or dependabot PRs will make it untrue. That's a bit unfortunate, but it's good to have a reminder this version is indirectly determining the version of OTel SDK dependencies.

Reply:
Yes, because of the issue described here: https://github.com/micrometer-metrics/tracing/pull/823

I put the comment there for this exact purpose: it reminds us to manually check the version that the new patch upgrade brings in and make sure we are conscious about it.

This also means that upgrading this with dependabot is a bit more work than doing it 100% manually. :)

I'm ok removing the comment to make things simpler but we need to be extra careful not merging OTel dependency upgrades in (not that the comment prevents this).

Reply:
I'm for leaving the comment. I'm just not confident we won't at some point miss it when a dependabot upgrade PR comes. We'll have to think how to work into the release process checking the version of OTel that gets resolved or something.